tox.ini

```diff
@@ -23,7 +23,7 @@ commands =
     black --diff --check --exclude "/(\.eggs|\.git|\.tox|\.venv|\.build|dist|charmhelpers|mod)/" .
 
 [testenv:unit]
-commands = pytest -s upgrade-scripts/003-upgrade-calico/test-upgrade-calico-cni.py
+commands = pytest -s scripts/upgrade-calico/test-upgrade-calico-cni.py
```

Comment:
nit; The test file could be moved under a `tests/unit` folder (to avoid clutter in the result snap build).